src/raylib.h

```diff
@@ -801,7 +801,8 @@ typedef enum {
     SHADER_LOC_MAP_BRDF,            // Shader location: sampler2d texture: brdf
     SHADER_LOC_VERTEX_BONEIDS,      // Shader location: vertex attribute: boneIds
     SHADER_LOC_VERTEX_BONEWEIGHTS,  // Shader location: vertex attribute: boneWeights
-    SHADER_LOC_BONE_MATRICES        // Shader location: array of matrices uniform: boneMatrices
+    SHADER_LOC_BONE_MATRICES,       // Shader location: array of matrices uniform: boneMatrices
+    SHADER_LOC_VERTEX_INSTANCE_TX   // Shader location: vertex attribute: instanceTransform
```

Comment:
Please check if this will conflict with the `MAX_MESH_VERTEX_BUFFERS` ([config.h#L260-L264](https://github.com/raysan5/raylib/blob/602583bcdbc46f317915c811a7e8e2416e65bbc7/src/config.h#L260-L264)).

Reply:
No it will not, that instances transform vbo is loaded/unloaded dynamically and its id is not stored inside the Mesh structure.